examples/examples-cpu/.saturn/start

```diff
@@ -0,0 +1,10 @@
+sudo apt update
+sudo apt-get install -y htop
+
+pip install \
+    --upgrade \
+        dask \
+        dask-saturn \
+        dask-xgboost==0.1.11 \
+        prefect>0.13.0 \
+        xgboost==0.90
```

Comment:
Is this just a "for now" start script? We are planning on building an examples-cpu image at some point right?

Reply:
short answer: yes

long answer: My goal is to get the structure to a place where I can test the changes on the Saturn side, and then to leave it to @rikturr to fill out this repo with the example content he wants to add. If that eventually means removing some or all or this start script and swapping in an image, I'm all for it

Reply:
To be clear, I still think that the concept of start script should always be supported in this project. But this specific one could probably be replaced by moving stuff into an image